app/src/main/java/com/celzero/bravedns/util/Constants.kt

```diff
@@ -122,13 +124,39 @@ class Constants {
         const val PORT_VAL_UNKNOWN = "unknown"
 
         const val NOTIFICATION_ACTION = "NOTIFICATION_VALUE"
-        const val VPN_NOTIFICATION_ACTION = "STOP_RETHINKDNS"
+        const val STOP_VPN_NOTIFICATION_ACTION = "RETHINK_STOP"
+        const val DNS_VPN_NOTIFICATION_ACTION = "RETHINK_DNSONLY" // battery-saver dns-only
+        const val DNS_FIREWALL_VPN_NOTIFICATION_ACTION = "RETHINK_FULLMODE" // default dns+firewall
+        const val RULES_FAILURE_NOTIFICATION_ACTION = "RETHINK_RULES_RELOAD" //
 
         const val NXDOMAIN = "NXDOMAIN"
 
-        const val WORK_MANAGER_TIMEOUT = 1*40*60*1000
+        val WORK_MANAGER_TIMEOUT = TimeUnit.MINUTES.toMillis(40)
 
         const val FAV_ICON_URL = "https://icons.duckduckgo.com/ip2/"
-    }
 
+        //Application theme constants
+        const val THEME_SYSTEM_DEFAULT = 0
+        const val THEME_LIGHT = 1
+        const val THEME_DARK = 2
+        const val THEME_TRUE_BLACK = 3
+
+        // Notification action buttons
+        const val NOTIFICATION_ACTION_STOP = 0
+        const val NOTIFICATION_ACTION_DNS_FIREWALL = 1
+        const val NOTIFICATION_ACTION_NONE = 2
+
+        //DNS MODES
+        const val APP_MODE_DNS = 0
+        const val APP_MODE_FIREWALL = 1
+        const val APP_MODE_DNS_FIREWALL = 2
+
+        //DNS TYPES
+        const val PREF_DNS_MODE_PROXY = 3
+        const val PREF_DNS_MODE_DNSCRYPT = 2
+        const val PREF_DNS_MODE_DOH = 1
+
+        val ACCESSIBILITY_SERVICE_HEARTBEAT_THRESHOLD_MS = TimeUnit.MINUTES.toMillis(5)
+        val NETWORK_REQUEST_WRITE_THRESHOLD_MS = TimeUnit.MINUTES.toMillis(1)
```

Comment:
add a comment, whatever this constant may be for.

Reply:
Added comment.

Reply:
Modified the comment.

Reply:
Modified the comment and the variable name.